### PR TITLE
JOSS Paper

### PR DESCRIPTION
**Pull request by tamasgal**
@Moelf and @aminnj I'd like to invite you to collaborate on a short (a few pages) JOSS paper, so that we get a DOI which is citable. The purpose of this publication is to introduce `UnROOT` to the world, outline the reason of its existence, show some short usage examples and compare it with existing other solution. No more, no less `;)`

Let me know if you are interested. I already added both of you to the template since we three are now definitely the main devs and the core team!

Feel free to dump ideas and add your affiliations (and @aminnj add your ORCID) if you're in.

### CODE REVIEW

**Comment by tamasgal** [comment]
For the last JOSS paper we used [hackmd.io](https://hackmd.io) but I have no premium account and it seems they limited some features since then `;)` However, three people can still work together, so that might be an option. Or do you have an other suggestion?

**Comment by aminnj** [comment]
Ah didn’t know that. Markdown is pretty wysiwyg so we can always use a shared google doc :)

**Comment by tamasgal** [comment]
Alright, I invited you to a shared Google doc, I guess that should do it `;)`

**Comment by aminnj** [comment]
CI fails because the table drawing characters don't exist for the font. When I tried https://whedon.theoj.org/, it just stripped out those characters entirely but still compiled:
[link](http://res.cloudinary.com/hkvhhndao/image/upload/v1634696728/x53lenj6hnjqej0yynpn.pdf)
Doesn't look bad, actually. I'll remove them from the md so that our CI works.

**Comment by Moelf** [comment]
is this ready to go?

**Review by Moelf — APPROVED** [review]

File: paper/paper.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+built-in abstract type for array-like objects, which allows to interface with
+array-routines from other packages easily, thanks to multiple dispatch, one of
+the main features of Julia.
+
+# Features and Functionality
+
+The `ROOT` dataformat is flexible and mostly self-descriptive. Users can define
+their own data structures (C++ classes) which derive from `ROOT` classes and
+serialise them into directories, trees and branches. The information about the
+deserialisation is written to the output file (therfore: self-descriptive) but
```
[the remark this answers is not in the document]
Comment (tamasgal) [reply paper/paper.md]: thanks!

**Comment by tamasgal** [comment]
[cg39hcpdd16nasx2zn2c.pdf](https://github.com/JuliaHEP/UnROOT.jl/files/8822072/cg39hcpdd16nasx2zn2c.pdf)

**Review by aminnj — APPROVED** [review]

**Pull request merged**